ArcGISRuntimeSDKQt_QMLSamples/CloudAndPortal/PortalUserInfo/PortalUserInfo.qml

```diff
@@ -0,0 +1,250 @@
+// [WriteFile Name=PortalUserInfo, Category=CloudAndPortal]
+// [Legal]
+// Copyright 2016 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick 2.6
+import QtQuick.Controls 1.4
+import QtQuick.Controls.Styles 1.4
+import Esri.ArcGISRuntime 100.0
+import Esri.ArcGISExtras 1.1
+import Esri.ArcGISRuntime.Toolkit.Dialogs 2.0
+
+Rectangle {
+    id: rootRectangle
+    clip: true
+
+    width: 500
+    height: 500
+
+    property real scaleFactor: System.displayScaleFactor
+    property var user
```

Comment:
Lines 32 thru 57 would make a nice snippet. Can you please tag it, start and end, with:

`//! [PortalUserInfo create portal]`

Thank you.